### PR TITLE
added new scipt builder functionalities

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/3900,  https://github.com/influxdata/docs-v2/issues/3889, https://github.com/influxdata/docs-v2/issues/3888

Rewrote Explore data with flux section to make it current:
- added load more feature
- added script builder warning message
-  choose time range on schema data

### CODE REVIEW

File: content/chronograf/v1.9/guides/querying-data.md [hunk 1]
```diff
@@ -48,33 +48,20 @@ For more information, see [InfluxQL support](/influxdb/cloud/query-data/influxql
...
-      |> toFloat()
-      |> percentile(percentile: 0.95)
-      |> group(except: ["_time", "_start", "_stop", "_value"])
-    ```
-    * Alternatively, you can enter your entire script manually.
-
-6. Click **Run Script** in the top bar of the **Script** pane. You can then preview your graph in the above pane.
-
-## Visualize your query
+1. Open the Data Explorer by clicking on **Explore** in the left navigation bar.
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/querying-data.md:51]:
```suggestion
1. Open the Data Explorer by clicking **Explore** in the left navigation bar.
```

File: content/chronograf/v1.9/guides/querying-data.md [hunk 2]
```diff
@@ -48,33 +48,20 @@ For more information, see [InfluxQL support](/influxdb/cloud/query-data/influxql
...
-    * Alternatively, you can enter your entire script manually.
-
-6. Click **Run Script** in the top bar of the **Script** pane. You can then preview your graph in the above pane.
-
-## Visualize your query
+1. Open the Data Explorer by clicking on **Explore** in the left navigation bar. 
+2. Select **Flux** as the source type.
+3. Click **Script Builder**.
+4. The **Schema**, **Script**, and **Flux Functions** panes appear.
+    - Use the **Schema** pane to explore your available data. Click the **+** sign next to a bucket name to expand its content.
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/querying-data.md:55]:
```suggestion
    - Use the **Schema** pane to explore your available data. Click the **{{< icon "plus" >}}** sign next to a bucket name to expand its content. 
```

File: content/chronograf/v1.9/guides/querying-data.md [hunk 3]
```diff
@@ -48,33 +48,20 @@ For more information, see [InfluxQL support](/influxdb/cloud/query-data/influxql
...
-
-## Visualize your query
+1. Open the Data Explorer by clicking on **Explore** in the left navigation bar. 
+2. Select **Flux** as the source type.
+3. Click **Script Builder**.
+4. The **Schema**, **Script**, and **Flux Functions** panes appear.
+    - Use the **Schema** pane to explore your available data. Click the **+** sign next to a bucket name to expand its content. 
+    - Use the **Script** pane to enter and view your Flux script.
+    - Use the **Flux Functions** pane to view details about the available Flux functions.
+5. To get started building a new script, click **Script Builder**. Using the Flux script builder, you can select a bucket, measurements and tags, fields and an aggregate function. Click **+Load More** to expand any truncated lists.  You can also choose a variety of time ranges on your schema data.
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/querying-data.md:58]:
```suggestion
5. To get started building a new script, click **Script Builder**. Using the Flux script builder, you can select a bucket, measurements and tags, fields and an aggregate function. Click **{{< icon "plus" >}} Load More** to expand any truncated lists.  You can also choose a variety of time ranges on your schema data.
```

File: content/chronograf/v1.9/guides/querying-data.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Explore data in Chronograf
-description: Query and visualize data in the Data Explorer.
+description: Query and visualize data in the Data Explorer
```
Comment (sanderson) [inline content/chronograf/v1.9/guides/querying-data.md:3]: We do want to keep the period here. This is used as the page description in search engine results.

**Review by sanderson — CHANGES_REQUESTED** [review]
Just a few minor changes.

File: content/chronograf/v1.9/guides/querying-data.md (hunk 4 above)
Comment (lwandzura) [reply content/chronograf/v1.9/guides/querying-data.md]: Done!  thanks for clarifying that

**Pull request merged**